### PR TITLE
Add calculator funcs

### PR DESCRIPTION
**Pull request by dospore**
### Motivation
- add functions which allow you to calculate the required inputs to achieve a position state

### Changes
- add calculator functions
- add calculator tests

TODO
- need to write some more edge case tests but this will do for now because ceebs

### CODE REVIEW

File: src/Accounting/index.ts [hunk 1]
```diff
@@ -1,8 +1,8 @@
 import { FlatOrder, FlatOrderWithSide } from "../Types/accounting";
 import { BigNumber } from 'bignumber.js';
 
-const RYAN_6 = new BigNumber(6); // a number accredited to our good friend Ryan Garner
-const LIQUIDATION_GAS_COST = new BigNumber (25); // When gas price is 250 gwei and eth price is 1700, the liquidation gas cost is 25 USD.
+export const RYAN_6 = new BigNumber(6); // a number accredited to our good friend Ryan Garner
+export const LIQUIDATION_GAS_COST = new BigNumber (25); // When gas price is 250 gwei and eth price is 1700, the liquidation gas cost is 25 USD.
```
Comment (raymogg) [inline src/Accounting/index.ts:5]: Should we have a config file of this per network or something? Obviously this varies a lot with gas price, we may almost need to have the function take in gas price and compute this based off gas usage and eth price?

**Review by raymogg — APPROVED** [review]
